### ISSUE TITLE
Hook / event for manipulate $data in outputSSO

### ISSUE THREAD

**Issue by timkelty** [nystudio107/vanillaforums#3]
Recently I had to hack this in: we wanted to use a custom field (`$currentUser->displayName`) as the `name` passed in the `outputSSO` `$data`.

Either a hook to manipulate that data, or an event (`vanillaforums.outputSSO` or something) to manipulate this data would be great. Of course if its an event, you'd have to pass `$data` by reference so it can be changed.

**Comment by khalwat** [comment]
Pull requests are always welcome :)

**Comment by timkelty** [comment]
Yep, planning on it. Just wanted to make the issues so I didn't forget.

Do you have a preference on event vs hook?

**Comment by khalwat** [comment]
I'd go with an event.

**Comment by timkelty** [comment]
good :) i'll whip something up
